src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -107,6 +107,7 @@ public static class ScriptConstants
         public const string XIdentityHeader = "X-IDENTITY-HEADER";
         public const string ContainerInstanceHeader = "fx-current-instance";
         public const string DynamicSku = "Dynamic";
+        public const string DynamicV2Sku = "DynamicV2";
```

Comment:
@balag0 - is this the right SKU constant?

Reply:
tagging @Tonewall as well .

Reply:
@mathewc  - @Tonewall - shared this offline -  https://github.com/Azure/azure-functions-host/blob/1f75688392491c60bae3411a2f50542dcbcf5b27/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L300 - please see if you can use the same helper

Reply:
Franklin confirmed to me offline that DynamicV2 is the SKU value that we'll see in WEBSITE_SKU and that's what I want to test for in my case.